### PR TITLE
Cu 860qfy4y3 carousel comp

### PR DESCRIPTION
**Pull request by TalMekler**

### CODE REVIEW

**Comment by TalMekler** [comment]
Task linked: [CU-860qfy4y3 Carousel Comp](https://app.clickup.com/t/860qfy4y3)

**Comment by TalMekler** [comment]
Task linked: [CU-860qfy4w4 Section Comp](https://app.clickup.com/t/860qfy4w4)

File: src/components/Carousel/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Style from "./styles.module.css";
+import "./carouselStyle.css";
+import RestaurantCard from "../RestaurantCard";
+import { ImArrowRight2, ImArrowLeft2 } from "react-icons/im";
+import AliceCarousel from "react-alice-carousel";
+import "react-alice-carousel/lib/alice-carousel.css";
+
+const RestaurantsCarousel = ({
+  restaurants = [
```
Comment (danielmargolin) [inline src/components/Carousel/index.js:10]: it's a bit strange to store mock data as a default for a component

File: src/components/Carousel/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          },
+        }}
+        infinite
+        autoPlay={autoPlay}
+        autoPlayInterval={interval}
+        // keyboardNavigation
+        disableDotsControls
+        autoHeight={false}
+        // disableButtonsControls
+        renderNextButton={()=> {return <div className="alice-carousel__next-btn-wrapper"><ImArrowRight2/></div>}}
```
Comment (danielmargolin) [inline src/components/Carousel/index.js:77]: ()=> <div className="alice-carousel__next-btn-wrapper"><ImArrowRight2/></div>

(redundant return)

**Pull request merged**